CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# How to Contribute
+
+Scope is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+# Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
+
+# Email and Chat
+
+The project uses the the scope-community email list and Slack:
+- Email: [scope-community](https://groups.google.com/forum/#!forum/scope-community)
+- Chat: Join the [Weave community](https://weaveworks.github.io/community-slack/) Slack workspace and use the [#scope](https://weave-community.slack.com/messages/scope/) channel
+
+Please avoid emailing maintainers found in the MAINTAINERS file directly. They
+are very busy and read the mailing lists.
```

Comment:
Maybe make the last sentence a bit friendlier? How about:

> It's usually best to mail the mailing list. The maintainers are usually quite busy and the mailing list will more easily find somebody who can reply quickly. Your question might also be the one others had on their mind.

Reply:
Sure!

Reply:
I actually don't think we should prioritize the mailing list over Slack though.